cluster/manifests/skipper/secret.yaml

```diff
@@ -7,4 +7,5 @@ metadata:
     application: skipper-ingress
 type: Opaque
 data:
-  lightstep-token: "{{ .ConfigItems.lightstep_token | base64 }}"
+  lightstep-token: "{{ .Cluster.ConfigItems.lightstep_token | base64 }}"
+  encryption-key: "{{ .Cluster.ConfigItems.skipper_ingress_encryption_key | base64 }}"
```

Comment:
You need a default value (empty?) for this, otherwise e2e will fail.

Reply:
true, ty

Reply:
fixed